monitoring/README.md

```diff
@@ -112,10 +112,16 @@ Operator and aggregates other helm charts such as Grafana and the Prometheus
 Node Exporter. Links to the charts and default values are included in the
 `user-values-prom-operator.yaml` file.
 
-**Note:** If you are using a cloud provider, you must use ingress, rather than
+**Note 1:** If you are using a cloud provider, you must use ingress, rather than
 NodePorts. Use the samples in the [samples/ingress](/samples/ingress)
 area of this repository to set up either host-based or path-based ingress.
 
+**Note 2:** Although the Grafana helm chart supports a value to set the initial
+admin password, you must use the `GRAFANA_ADMIN_PASSWORD` value in
+`$USER_DIR/monitoring/user.env` to set this value. If not specified, the
+initial Grafana admin password will be randomly generated and logged during the
+initial deployment of the monitoring components.
+
```

Comment:
What do you mean by "not specified" in user.env? Does that mean the user doesn't include the GRAFANA_ADMIN_PASSWORD line in their user.env at all? If they keep the line that's in the repo user.env but don't change it, would their password be "yourPasswordHere"?

I'm also not sure what you mean by the password is "logged" during deployment. Logged where? How will they find it?

Reply:
Not specified in the environment variable sense - whether that's through $USER_DIR/user.env, $USER_DIR/monitoring/user.env, or just the terminal environment.